### ISSUE TITLE
Getting a Segmentation fault when building site on CircleCI

### ISSUE THREAD

**Issue by jameswlane** [cibuilds/hugo#88]
When building the site on CircleCI I am getting a `Segmentation fault` error.

https://circleci.com/gh/jameswlane/cantyouju.st/9

https://github.com/jameswlane/cantyouju.st

**Comment by felicianotech** [comment]
I haven't seen that before. I'm looking into it.

**Comment by felicianotech** [comment]
There's something wrong with your SASS imports. When I disable them, Hugo runs fine.

**Comment by felicianotech** [comment]
Okay so I commented out the imports and Hugo was happy. I started adding them back one by one. Hugo didn't segfault until I re-added this line:

> @import "../../node_modules/foundation-sites/scss/foundation";

Something that's happening in there is what's causing Hugo to segfault for some reason.

**Comment by jameswlane** [comment]
Awesome, I will do some digging into that.. and let you know. Thanks for looking at this so quickly.

**Comment by jameswlane** [comment]
@felicianotech Strange you are right removing it allows it to build. Now I need to try and figure out why Foundation SCSS library causes it to break.

Thanks again for your help!

**Comment by raww** [comment]
I'm actually struggling with this - easily reproduced with the hugo-fresh theme.

**Comment by felicianotech** [comment]
@raww @shanev Investigating....

**Comment by felicianotech** [comment]
I'm going to have to custom compile Hugo in order to fix this issue. It'll be fixed by Sunday.

**Comment by raww** [comment]
@felicianotech awesome thanks!

**Comment by felicianotech** [comment]
@raww I'm a day (almost two) late but here we go. Can you test the latest Hugo image and tell me if it's working for you? You can use any of these three tags to test:

```
cibuilds/hugo:latest
cibuilds/hugo:0.55
cibuilds/hugo:0.55.5
```

**Comment by paumel** [comment]
@felicianotech latest image doesn't work for me because it's not extended

**Comment by felicianotech** [comment]
@paumel Hmm, that shouldn't be the case. What's the exact error that you're getting?

**Comment by raww** [comment]
@felicianotech `cibuilds/hugo:latest` works for me now. 👍

**Comment by felicianotech** [comment]
Really? I didn't do anything since yesterday. Are these CircleCI builds? If so, and if they're public, can I see?

If they aren't public and you don't mind me taking a look, you can email me ricardo@circleci.com

I would love to understand what happened so that you don't have an issue again later today or tomorrow or something random like that.

**Comment by raww** [comment]
Yeah, I only just got around to testing the new tag from yesterday.

**Comment by felicianotech** [comment]
Okay this is officially fixed (as far as I can tell) in #105.

**Comment by felicianotech** [comment]
For the record for anyone looking at this, starting from the `cibuilds/hugo:0.55.5` image. Older versions of Hugo/this image won't include this fix. Newer versions, such as the upcoming Hugo v0.56 release, will.
